src/components/cl/hier/allgatherv/allgatherv.h

```diff
@@ -17,7 +17,7 @@ enum
 extern ucc_base_coll_alg_info_t
     ucc_cl_hier_allgatherv_algs[UCC_CL_HIER_ALLGATHERV_ALG_LAST + 1];
 
-#define UCC_CL_HIER_ALLGATHERV_DEFAULT_ALG_SELECT_STR "allgatherv:0-2k:host:@gab"
+#define UCC_CL_HIER_ALLGATHERV_DEFAULT_ALG_SELECT_STR "allgatherv:0-7864320:host:@gab"
```

Comment:
This might not be a sane default, especially for smaller teams.

Reply:
echoing my comment here, instead of 7864320, just maybe put 8mb to be more intuitive